src/components/qr-modal/qr-modal.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { View, Modal } from 'react-native';
+import { CameraKitCameraScreen, CameraKitCamera } from 'react-native-camera-kit';
```

Comment:
this is failing for web, since we won't have support for qrcode scan on web just create a new qr-modal.web.tsx and put a dummy empty component. It should solve the problem,